src/voice/voice-quota.service.ts

```
@@ -16,41 +16,38 @@ export class VoiceQuotaService {
+            },
         });
 
         if (!user) return false;
 
-        // Determine plan
-        const activeSubscription = user.subscriptions.find(
-            (s) => s.status === SUBSCRIPTION_STATUS.ACTIVE && s.endsAt && s.endsAt > new Date()
-        );
-        const isPremium = !!activeSubscription;
-
+        // Determine plan from included subscription
```

ESLint

[error] Delete `····`

(prettier/prettier)